### PR TITLE
cmake: use standard CMAKE_PREFIX_PATH way of specifying where to find a custom Qt5 installation

### PR DESCRIPTION
**Pull request by marco-m**
Addresses #190 (Move QTDIR to cmake variable?)

Tested on OS X. Let's see if travis and appveyor are happy...

 I found some other things that can be improved but I will propose them in separate PRs.

### CODE REVIEW

**Comment by sieren** [comment]
Except for that one comment LGTM. I'll merge if CI goes green. Thanks!

**Comment by sieren** [comment]
PS: Might need to rebase against master again, just merged the other PR

**Comment by sieren** [comment]
Thanks!

File: CMakeLists.txt [hunk 1]
```diff
@@ -2,40 +2,21 @@ cmake_minimum_required(VERSION 3.2)
...
 # Instruct CMake to run moc automatically when needed.
 set(CMAKE_AUTOMOC ON)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
-# Find the QtWidgets library
-find_package(Qt5Widgets REQUIRED)
-find_package(Qt5Network)
-find_package(Qt5PrintSupport)
+find_package(Qt5 5.5 COMPONENTS Widgets Network PrintSupport)
```
Comment (marco-m) [inline CMakeLists.txt]: that line requires Qt to be at least 5.5. I choose 5.5 based on what was in the travis configuration file.

Comment (sieren) [inline CMakeLists.txt]: Ah cool, so its a minimum requirement. Then it's alright 👍

**Pull request merged**